Runtime/Core/CinemachineDebug.cs

```diff
@@ -17,6 +17,7 @@ public static void ReleaseScreenPos(Object client)
                 mClients.Remove(client);
         }
 
+#if CINEMACHINE_UNITY_IMGUI
```

Comment:
Need at the top:
#if !UNITY_2019_3_OR_NEWER
#define CINEMACHINE_UNITY_IMGUI
#endif